AIX build broken

Recently OpenBLAS build in AIX broke with the below error. Looks like these are coming from recently added code., 

/opt/IBM/openxlC/17.1.2/bin/ibm-clang_r -O2 -DSMALL_MATRIX_OPT -DGEMM_GEMV_FORWARD -DSBGEMM_GEMV_FORWARD -DMAX_STACK_ALLOC=2048 -fopenmp -DHAVE_P10_SUPPORT -Wall -DF_INTERFACE_IBM -fPIC -DDYNAMIC_ARCH -DSMP_SERVER -DUSE_OPENMP -DNO_WARMUP -DMAX_CPU_NUMBER=16 -DMAX_PARALLEL_NUMBER=1 -DBUILD_BFLOAT16 -DBUILD_SINGLE=1 -DBUILD_DOUBLE=1 -DBUILD_COMPLEX=1 -DBUILD_COMPLEX16=1 -DVERSION=\"0.3.30.dev\" -DUSE_OPENMP -fopenmp -fno-integrated-as -m64 -UASMNAME -UASMFNAME -UNAME -UCNAME -UCHAR_NAME -UCHAR_CNAME -DASMNAME= -DASMFNAME=_ -DNAME=_ -DCNAME= -DCHAR_NAME=\"_\" -DCHAR_CNAME=\"\" -DNO_AFFINITY -I.. -std=c11 -Wall -Werror -Wl,-blibpath:/usr/lib:/lib -DIBFLOAT16 -DOBFLOAT16 -o test_bgemv compare_sgemv_bgemv.c

```
compare_sgemv_bgemv.c:122:51: error: format specifies type 'int' but the argument has type 'blasint' (aka 'long') [-Werror,-Wformat]
  121 |                 printf("Mismatch at trans=%c, alpha=%.2f, beta=%.2f, i=%d, j=%d, k=%d: CC=%.6f, C=%.6f\n",
      |                                                                                    ~~
      |                                                                                    %ld
  122 |                        transA, alpha, beta, i, j, k, float16to32(CC[j << l]), truncate_float32_to_bfloat16(C[j << l]));
      |                                                   ^
compare_sgemv_bgemv.c:128:51: error: format specifies type 'int' but the argument has type 'blasint' (aka 'long') [-Werror,-Wformat]
  127 |                 printf("Mismatch at trans=%c, alpha=%.2f, beta=%.2f, i=%d, j=%d, k=%d: CC=%.6f, C=%.6f\n",
      |                                                                                    ~~
      |                                                                                    %ld
  128 |                        transA, alpha, beta, i, j, k, float16to32(CC[j << l]), truncate_float32_to_bfloat16(DD[j]));
      |                                                   ^
```

/opt/IBM/openxlC/17.1.2/bin/ibm-clang_r -O2 -DSMALL_MATRIX_OPT -DGEMM_GEMV_FORWARD -DSBGEMM_GEMV_FORWARD -DMAX_STACK_ALLOC=2048 -fopenmp -DHAVE_P10_SUPPORT -Wall -DF_INTERFACE_IBM -fPIC -DDYNAMIC_ARCH -DSMP_SERVER -DUSE_OPENMP -DNO_WARMUP -DMAX_CPU_NUMBER=16 -DMAX_PARALLEL_NUMBER=1 -DBUILD_BFLOAT16 -DBUILD_SINGLE=1 -DBUILD_DOUBLE=1 -DBUILD_COMPLEX=1 -DBUILD_COMPLEX16=1 -DVERSION=\"0.3.30.dev\" -DUSE_OPENMP -fopenmp -fno-integrated-as -m64 -UASMNAME -UASMFNAME -UNAME -UCNAME -UCHAR_NAME -UCHAR_CNAME -DASMNAME= -DASMFNAME=_ -DNAME=_ -DCNAME= -DCHAR_NAME=\"_\" -DCHAR_CNAME=\"\" -DNO_AFFINITY -I.. -std=c11 -Wall -Werror -Wl,-blibpath:/usr/lib:/lib -DIBFLOAT16 -DOBFLOAT16 -o test_bgemm compare_sgemm_bgemm.c

```
compare_sgemm_bgemm.c:129:27: error: format specifies type 'int' but the argument has type 'blasint' (aka 'long') [-Werror,-Wformat]
  128 |             printf("Mismatch at i=%d, j=%d, k=%d: CC=%.6f, C=%.6f\n",
      |                                               ~~
      |                                               %ld
  129 |                     i, j, k, float16to32(CC[i * m + j]), truncate_float32_to_bfloat16(C[i * m + j]));
      |                           ^
compare_sgemm_bgemm.c:135:27: error: format specifies type 'int' but the argument has type 'blasint' (aka 'long') [-Werror,-Wformat]
  134 |             printf("Mismatch at i=%d, j=%d, k=%d: CC=%.6f, DD=%.6f\n",
      |                                               ~~
      |                                               %ld
  135 |                     i, j, k, float16to32(CC[i * m + j]), truncate_float32_to_bfloat16(DD[i * m + j]));
      |                           ^
```

--- a/interface/lapack/trtri.c
+++ b/interface/lapack/trtri.c
@@ -127,10 +127,10 @@ int NAME(char *UPLO, char *DIAG, blasint *N, FLOAT *a, blasint *ldA, blasint *In
 #endif
 
 #ifdef SMP
-if (args.n <= 150)
-  args.nthreads = 1;
-else
-  args.nthreads = num_cpu_avail(4);
+  if (args.n <= 150)
+    args.nthreads = 1;
+  else
+    args.nthreads = num_cpu_avail(4);
 
   if (args.nthreads == 1) {
 #endif

--- a/test/compare_sgemm_bgemm.c
+++ b/test/compare_sgemm_bgemm.c
@@ -125,14 +125,18 @@ main (int argc, char *argv[])
                 float16to32 (AA[k * j + l]) * float16to32 (BB[i + l * n]);
             }
           if (!is_close(float16to32(CC[i * m + j]), truncate_float32_to_bfloat16(C[i * m + j]), 0.01, 0.001)) {
-            printf("Mismatch at i=%d, j=%d, k=%d: CC=%.6f, C=%.6f\n",
+#ifdef DEBUG
+            printf("Mismatch at i=%d, j=%d, k=%ld: CC=%.6f, C=%.6f\n",
                     i, j, k, float16to32(CC[i * m + j]), truncate_float32_to_bfloat16(C[i * m + j]));
+#endif
             ret++;
           }
 
           if (!is_close(float16to32(CC[i * m + j]), truncate_float32_to_bfloat16(DD[i * m + j]), 0.0001, 0.00001)) {
-            printf("Mismatch at i=%d, j=%d, k=%d: CC=%.6f, DD=%.6f\n",
+#ifdef DEBUG
+            printf("Mismatch at i=%d, j=%d, k=%ld: CC=%.6f, DD=%.6f\n",
                     i, j, k, float16to32(CC[i * m + j]), truncate_float32_to_bfloat16(DD[i * m + j]));
+#endif
             ret++;
           }
             

--- a/test/compare_sgemv_bgemv.c
+++ b/test/compare_sgemv_bgemv.c
@@ -100,7 +100,7 @@ int main(int argc, char *argv[])
             SGEMV(&transA, &x, &x, &alpha, A, &x, B, &k, &beta, C, &k);
             BGEMV(&transA, &x, &x, &alpha_bf16, AA, &x, BB, &k, &beta_bf16, CC, &k);
 
-            for (int i = 0; i < x; i++)
+            for (i = 0; i < x; i++)
               DD[i] *= beta;
 
             for (j = 0; j < x; j++)
@@ -118,14 +118,18 @@ int main(int argc, char *argv[])
             {
               if (!is_close(float16to32(CC[j << l]), truncate_float32_to_bfloat16(C[j << l]), 0.01, 0.001))
               {
-                printf("Mismatch at trans=%c, alpha=%.2f, beta=%.2f, i=%d, j=%d, k=%d: CC=%.6f, C=%.6f\n",
+#ifdef DEBUG
+                printf("Mismatch at trans=%c, alpha=%.2f, beta=%.2f, i=%d, j=%d, k=%ld: CC=%.6f, C=%.6f\n",
                        transA, alpha, beta, i, j, k, float16to32(CC[j << l]), truncate_float32_to_bfloat16(C[j << l]));
+#endif
                 ret++;
               }
               if (!is_close(float16to32(CC[j << l]), truncate_float32_to_bfloat16(DD[j]), 0.001, 0.0001))
               {
-                printf("Mismatch at trans=%c, alpha=%.2f, beta=%.2f, i=%d, j=%d, k=%d: CC=%.6f, C=%.6f\n",
+#ifdef DEBUG
+                printf("Mismatch at trans=%c, alpha=%.2f, beta=%.2f, i=%d, j=%d, k=%ld: CC=%.6f, C=%.6f\n",
                        transA, alpha, beta, i, j, k, float16to32(CC[j << l]), truncate_float32_to_bfloat16(DD[j]));
+#endif
                 ret++;
               }
             }

